--- a/pipeline_definition.py	2023-11-09 13:36:35
+++ b/pipeline_definition.py	2023-11-07 20:07:49
@@ -265,6 +265,9 @@
         if parameter_class is None:
             return None  # runtime type does not support parameters, skip
 
+        if not ElyraProperty.subclass_exists_for_property(parameter_class.property_id):
+            ElyraProperty.build_property_map()
+
         # Convert pipeline parameters to runtime-specific instances
         converted_value = ElyraProperty.create_instance(parameter_class.property_id, self.pipeline_parameters)
         if converted_value is not None:
